ArcadiaParties.API/Startup.cs

```diff
@@ -43,8 +46,8 @@ public void ConfigureServices(IServiceCollection services)
                 .AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
                 .AddJwtBearer(options =>
                 {
-                    options.Audience = "a2ccb221-60e2-47b8-b28c-bf88a59f7f4a";
-                    options.MetadataAddress = "https://login.microsoftonline.com/fa4e9c1f-6222-443d-a083-28f80c1ffefc/.well-known/openid-configuration";
+                    options.Audience = OAuthSettings.ClientId;
+                    options.MetadataAddress = "https://login.microsoftonline.com/" + OAuthSettings.Tenant + "/.well-known/openid-configuration";
```

Comment:
I'd make it another way - keep urls themselves in the configuration.

Reply:
I've moved whole urls to secret